### PR TITLE
CUMULUS-1170 Use npm instead of yarn

### PR DESCRIPTION
**Pull request by yjpa7145**

### CODE REVIEW

File: docs/deployment/README.md [hunk 1]
```diff
@@ -360,7 +360,8 @@ Please note that the requirements are similar to the [Cumulus stack deployment r
 
 ### Install dashboard
 
-To install the dashboard clone the Cumulus-dashboard repository into the root deploy directory and install dependencies with `yarn install`:
+To install the dashboard clone the Cumulus-dashboard repository into the root
```
Comment (markdboyd) [inline docs/deployment/README.md]: We still have to use `yarn` for the dashboard

File: README.md [hunk 2]
```diff
@@ -35,26 +34,31 @@ nvm install
...
+    $ npm install -g lerna
 
 ### Install Local Dependencies
 
-We use yarn for local package management
+We use npm for local package management
 
-    $ yarn install
-    $ yarn ybootstrap
+    $ npm ci
```
Comment (markdboyd) [inline README.md]: Why would we use `npm ci` for local development?

Comment (markdboyd) [reply README.md]: I know that `npm ci` is much more deterministic based on the lockfile, I'm just not used to seeing it used for local development. But I can't really think of a reason why it shouldn't be.

File: README.md [hunk 3]
```diff
@@ -44,7 +49,7 @@ We use Lerna to manage multiple Cumulus packages in the same repo. You need to i
 
 We use npm for local package management
 
-    $ npm install
+    $ npm ci
     $ npm run bootstrap
```
Comment (mennovandiermen) [inline README.md]: We should be able to update `bootstrap` commands to use `ci` with the `--ci` flag, see https://github.com/mndewitt/lerna/blob/master/README.md#--ci

**Pull request closed without merging**